File: Iniciante/URI 1050.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var DDD = parseInt(lines.shift());
switch (DDD) {
    case 61: console.log('Brasilia'); break;
    case 71: console.log('Salvador'); break;
    case 11: console.log('Sao Paulo'); break;
    case 21: console.log('Rio de Janeiro'); break;
    case 32: console.log('Juiz de Fora'); break;
    case 19: console.log('Campinas'); break;
    case 27: console.log('Vitoria'); break;
    case 31: console.log('Belo Horizonte'); break;
    default: console.log('DDD nao cadastrado');
}

File: Iniciante/URI 1050.pas
program uri_1050;
uses crt;
const d61 = 'Brasilia';
	  d71 = 'Salvador';
	  d11 = 'Sao Paulo';
	  d21 = 'Rio de Janeiro';
	  d32 = 'Juiz de Fora';
	  d19 = 'Campinas';
	  d27 = 'Vitoria';
	  d31 = 'Belo Horizonte';
var n : integer; 
	ddd: string;
BEGIN
	readln(n);
	case n of
	61: ddd:= d61;
	71: ddd:= d71;
	11: ddd:= d11;
	21: ddd:= d21;
	32: ddd:= d32;
	19: ddd:= d19;
	27: ddd:= d27;
	31: ddd:= d31;
	else ddd:='DDD nao cadastrado'
	end;
	writeln(ddd);
	
END.
